problem-json/build.gradle.kts

```diff
@@ -1,13 +1,14 @@
 plugins {
-    id("io.micronaut.build.internal.module")
+    id("io.micronaut.build.internal.problem-json-module")
 }
 
 dependencies {
-    annotationProcessor(mn.micronaut.serde.processor)
+    annotationProcessor(mnSerde.micronaut.serde.processor)
     api(libs.managed.zalando.problem)
-    implementation(mn.micronaut.serde.api)
+    implementation(mnSerde.micronaut.serde.api)
     implementation(mn.micronaut.validation)
     implementation(mn.micronaut.http.server)
+    implementation(mn.micronaut.jackson.databind)
```

Comment:
why is this necessary? are not we using serde?

Reply:
I fixed the dependencies, but now I get a test failure in `test-suite`. Investigating.

Reply:
Just had a quick look...  When we switch to Serde, we do not get a `ConstraintViolationException` anymore (ie: in `ConstraintViolationProblemSpec`), we get an error:

```
io.micronaut.core.convert.exceptions.ConversionErrorException: Failed to convert argument [contact] for value [null] due to: Unable to deserialize type [io.micronaut.problem.ConstraintViolationProblemSpec$ContactCreateForm]. Required property [String name] is not present in supplied data
```

This means `ProblemConstraintViolationExceptionHandler` doesn't get invoked

Needs more thought... 🤔

Reply:
@timyates yeah, we have three tests failing now, and I think at least two are the problem you mention.

Reply:
Fixed it here: https://github.com/micronaut-projects/micronaut-problem-json/pull/212/commits/3de7e529b7191ea57eadaa301bdd4dbd0e4bdf8e